specification/servicefabricmanagedclusters/ServiceFabricManagedClusters.Management/NodeType.tsp

```diff
@@ -1172,6 +1177,29 @@ union SecurityType {
    * Standard is the default security type for all machines.
    */
   Standard: "Standard",
+
+  /**
+   * ConfidentialVM provides hardware-based encryption, trusted execution environment (TEE) and isolation of the VM memory & CPU from host.
+   */
+  ConfidentialVM: "ConfidentialVM",
+}
+
+/**
+ * Specifies the securityEncryptionType type of the nodeType. Only DiskWithVMGuestState and VMGuestStateOnly are currently supported
+ */
+union SecurityEncryptionType {
+  string,
+
+  /**
+   * For encryption of the managed disk along with VMGuestState blob.
+   */
+  TrustedLaunch: "DiskWithVMGuestState",
```

Comment:
is the name and value diff intentional?

Reply:
https://msazure.visualstudio.com/One/_git/winfab-RP?path=/src/sfmc/SfmcBackendService/NodeType/DataAccess/SecurityEncryptionType.cs&_a=contents&version=GBdevelop

Reply:
copy pase issue. fixed